definitions/variable/emissions/carbon-management.yaml

```diff
@@ -0,0 +1,97 @@
+        direct air carbon capture and storage (DACCS), and other CO2 removal (exclusive of land-use change) in these sectors
+    unit: Mt CO2/yr
+
+- Carbon Capture|Geologic Storage:
+    description: Captured carbon dioxide (CO2) that is injected into subsurface geologic storage reservoirs (e.g., in situ mineralization).
+    unit: Mt CO2/yr
+    note: Captured CO2 that is reacted at the earth's surface (e.g., ex situ mineralization) should be reported under Carbon Capture|Other Storage
+
+- Carbon Capture|Leakage:
+    description: Captured carbon dioxide (CO2) that is subsequently vented, leaked, or reemitted during or after transmission to storage repositories
+    unit: Mt CO2/yr
+    note: Leakage of CO2 stored in previous years should be reported here
+    
+-Carbon Capture|Other Storage:
+    description: Captured carbon dioxide (CO2) that is stored in reservoirs not elsewhere specified
+    unit: Mt CO2/yr
+
+-Carbon Capture|Utilization:
+    description: Captured carbon dioxide (CO2) that used in the manufacture of fuels, materials, or to enhance the productivity of oil and/or natural gas reservoirs.
+    unit: Mt CO2/yr
+
+-Carbon Capture|Utilization|Durable Storage:
+    description: Captured carbon dioxide (CO2) that is used in the manufacture durable materials (e.g., cement), 
+        or to enhance the productivity of oil and/or natural gas reservoirs wherein the captured carbon is stored on centenial or longer timescales. 
+    unit: Mt CO2/yr
+    
+-Carbon Capture|Utilization|Short-lived Storage:
+    description: Captured carbon dioxide (CO2) that is used in the manufacture of fuels or short-lived materials 
+        wherin the captured carbon is reemitted to the atmosphere upon consumption of these products or within annual to decadal timescales. 
+    unit: Mt CO2/yr
+
+# Carbon Removal reports the method of CDR in level-1, with additional disaggration as to the fate of removed carbon for materials in Level-2
+
+- Carbon Removal:
+    description: Gross removals of CO2 from the atmosphere through deliberate human activities
+    unit: Mt CO2/yr
+
+- Carbon Removal|Biochar:
+    description: production of biochar using pyrolysis
+    unit: Mt CO2/yr
+
+- Carbon Removal|Direct Air Capture:
+    description: CO2 captured directly from the atmosphere in solvent, sorbent, or other chemical processes, excluding any fossil process heat.
+    unit: Mt CO2/yr
+
+- Carbon Removal|Direct Ocean Capture:
+    description: dissolved inorganic carbon that is electrochemically separated from seawater and stored in geologic reservoirs or other long-term storage
+    unit: Mt CO2/yr
+
+- Carbon Removal|Terrestrial Enhanced Weathering:
+    description: Carbon dioxide (CO2) removed from the atmosphere by spreading ground silicate bearing rock 
+        over croplands or terrestrial ecosystems, thereby converting aqueous CO2 into solid carbonate minerals or ocean alkalinity
+    unit: Mt CO2/yr
+
+- Carbon Removal|Ocean:
```

Comment:
The distinction between the two variables "Carbon Removal|Direct Ocean Capture" and "Carbon Removal|Ocean" is really not obvious, please propose more intuitive variable names.

Maybe "Direct Ocean Capture" should be a sub-category similar to "Ocean|Alkalinization"?

Reply:
Yes, that would make sense to me, to have "Ocean" as the top category, and then add sub-categories as Alkalinization and Direct Ocean Capture

Reply:
Agreed, just implemented this